eng/common/pipelines/templates/jobs/archetype-sdk-tests-generate.yml

```diff
@@ -1,6 +1,9 @@
 parameters:
 - name: AdditionalParameters
   type: object
+- name: DependsOn
+  type: object
+  default: null
```

Comment:
Is null a value value? I don't see it mentioned anywhere at https://docs.microsoft.com/en-us/azure/devops/pipelines/process/templates?view=azure-devops.  Should we just leave this as empty instead?

Reply:
The challenge I'm having is that leaving it empty marks it as a required parameter, but defaulting it to `[]` means my `${{ if ... }}` conditional won't work. Perhaps there's a better way?

Reply:
But to your main question, it does work when empty. My understanding is it's a yaml object type, which I assume is a JSON object type, for which null is a valid field.

Reply:
As long as it works in your testing I'm fine with this approach I just haven't seen it before and couldn't find any reference to null in the docs.